controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -460,6 +460,12 @@ public OperationContext createContext(String scope, String name) {
         return withCompletion(getStream(scope, stream, context).getActiveTxns(), executor);
     }
 
+    @Override
+    public CompletableFuture<List<ScaleIndicent>> getScaleIncidents(final String scope, final String name,
```

Comment:
Because ScaleEvent is an object in our codebase which refers to event posted in the request stream. 
So far we have been calling it 'scale event' only in our conversations and there is no specific name for the record in the metadata. 
So we need to give it a name that is disambiguated and differentiated with ScaleEvent. 
I could have called it scale-record. However, we have used the noun 'record' in metadata store context to represent a single row in our metadata tables a la SegmentRecord, HistoryRecord and IndexRecord. 

Hence for the want of better name i called it scaleIncident (though incident has a bit of negative connotation, so please suggest a better name.)